values.yaml

```diff
@@ -257,6 +257,9 @@ security:
     # Note: The user is created regardless of if the existingSecret is provided or not.
     username: provisioner
     password: iHaveSuperpowers
+    # Bcrypt-hashed password
```

Comment:
Could expand the description a bit 

> Bcrypt-hashed password for the provisioner user.

Also, see that I've mentioned the format in `security.admin.initialPassword`. Could do that here as well

Reply:
done